components/neeva/browser/content_filter_rules_config.cc

```diff
@@ -37,6 +43,17 @@ ContentFilterRulesConfig* ContentFilterRulesConfig::GetOrCreate(
 }
 
 void ContentFilterRulesConfig::EnableRulesFile(const std::string& file) {
+  bool is_valid_file = false;
+  for (const auto* valid_file : kValidRulesFiles) {
+    if (file == valid_file) {
+      is_valid_file = true;
+      break;
+    }
+  }
+  if (!is_valid_file) {
+    LOG(ERROR) << "Unexpected rules file: " << file;
```

Comment:
why don't you return early if the file is invalid?

Reply:
Oh yes, that would be better. Not critical as it will fail later on, but I like returning early too. Thanks for catching that.